src/Image.php

```diff
@@ -109,7 +122,7 @@ public function save($outputPath = '')
 
         $this->addFormatManipulation($outputPath);
 
-        GlideConversion::create($this->pathToImage)
+        GlideConversion::create($this->pathToImage, self::$configurations)
```

Comment:
Is there a reason why `$configurations` is static?

Reply:
The idea was that it allows to just set the tempdir once, and all successive calls to Image::load will use this directory.
That would make it easier to set this path if "Image" is used from other libraries (for example laravel-medialibrary).

Other than that : Thanks for the review. I will try again.